### PR TITLE
Keylight get-user-by-id command

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23358

## Description
add a command to get the details of a user by his ID. Specifically his Email.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.py [hunk 1]
```diff
@@ -632,7 +640,8 @@ def main():
         'kl-get-record-attachment': get_record_attachment_command,
         'kl-get-record-attachments': get_record_attachments_command,
         'kl-delete-record-attachment': delete_record_attachment_command,
-        'fetch-incidents': fetch_incidents
+        'kl-get-user-by-id': get_user_by_id_command,
+        'fetch-incidents': fetch_incidents,
```
Comment (yaakovi) [inline Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.py:644]: thanks for the comma 👍

**Comment by yaakovi** [comment]
@kirbles19 please review:
* yml changes (only the new command)
* readme
*changelog

**Comment by orlichter1** [comment]
Update: We seem to have lost permission to do an update on records. This fails the test. In touch with the instance providers in order to fix this

File: Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.yml [hunk 2]
```diff
@@ -555,6 +568,51 @@ script:
...
+      description: The user's username.
+      type: String
+    - contextPath: Keylight.User.Active
+      description: Is the user active.
+      type: Boolean
+    - contextPath: Keylight.User.FirstName
+      description: The user's first name.
+      type: String
+    - contextPath: Keylight.User.MiddleName
+      description: The users middle name.
```
Comment (kirbles19) [inline Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.yml:599]: The user's middle name.

File: Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.yml [hunk 3]
```diff
@@ -555,6 +568,51 @@ script:
...
+      type: String
+    - contextPath: Keylight.User.LastName
+      description: The user's last name.
+      type: String
+    - contextPath: Keylight.User.EmailAddress
+      description: The user's email address.
+      type: String
+    - contextPath: Keylight.User.HomePhone
+      description: The user's home phone.
```
Comment (kirbles19) [inline Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.yml:608]: The user's home phone number.

File: Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.yml [hunk 4]
```diff
@@ -555,6 +568,51 @@ script:
...
+      type: String
+    - contextPath: Keylight.User.WorkPhone
```
Comment (kirbles19) [inline Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.yml:610]: The user's work phone number.

File: Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.yml [hunk 5]
```diff
@@ -555,6 +568,51 @@ script:
...
+      description: The user's work phone.
+      type: String
+    - contextPath: Keylight.User.MobilePhone
```
Comment (kirbles19) [inline Packs/Lokpath_Keylight/Integrations/Lockpath_KeyLight_v2/Lockpath_KeyLight_v2.yml:613]: The user's mobile phone number.

**Comment by kirbles19** [comment]
@orlichter1 Done.

**Pull request merged**